idb/postgres/internal/encoding/types.go

```diff
@@ -1,41 +1,27 @@
 package encoding
 
 import (
-	"github.com/algorand/indexer/types"
-
 	sdk "github.com/algorand/go-algorand-sdk/v2/types"
 )
 
+// AlgodEncodedAddress is an address encoded in the format used by algod.
+type AlgodEncodedAddress sdk.Address
+
 type blockHeader struct {
 	sdk.BlockHeader
-	BranchOverride      sdk.Digest `codec:"prev"`
-	FeeSinkOverride     sdk.Digest `codec:"fees"`
-	RewardsPoolOverride sdk.Digest `codec:"rwd"`
+	ExpiredParticipationAccountsOverride []AlgodEncodedAddress `codec:"partupdrmv"`
```

Comment:
why only this Address field need to be AlgodEncodedAddress but the others can be sdk.Address?